.github/workflows/dfetch.yml

```diff
@@ -5,6 +5,10 @@ on: push
 jobs:
   dfetch:
     runs-on: ubuntu-latest
+    permissions:
```

Comment:
@spoorcc : Apparently, these permissions are required now.

Reply:
mmm, interesting, hope to solve it one day with https://github.com/dfetch-org/dfetch/issues/361